script.js

```diff
@@ -0,0 +1,247 @@
+  popupDescription.innerHTML = data.description;
+  productTitle.innerHTML = data.name;
+  popupPosition.appendChild(loadPosition(data.items));
+  imageSource.src = data.image;
+  data.technologies.forEach((element) => {
+    const li = document.createElement('li');
+    const button = document.createElement('button');
+    button.classList.add('button-category');
+    button.innerHTML = element;
+    li.appendChild(button);
+    popupTechnology.appendChild(li);
+  });
+  modal.style.display = 'block';
+
+  // redirect either to github source or to live demo
+  liveLinkBtn.addEventListener('click', () => {
+    redirectToLink('https://peter-mwovi.github.io/portfolio_setup/');
+  });
+  githubLinkBtn.addEventListener('click', () => {
+    redirectToLink('https://github.com/peter-mwovi/portfolio_setup');
+  });
+
+  // When the user clicks on <span> (x), close the modal
+  span.onclick = function () {
+    modal.style.display = 'none';
+  };
+}
+
+function buildProjectCard(data) {
+  const workSection = document.querySelector('.works-section');
+  data.forEach((cardData) => {
+    /* A Single row that represents the project's card */
+    const rows = document.createElement('article');
+    rows.className = 'div-works';
+    // Project's image-side
+    const work1bImg = document.createElement('img');
+    work1bImg.className = 'article_img';
+    work1bImg.src = cardData.image;
+    // Project's details-side
+    const work1b = document.createElement('div');
+    const h2 = document.createElement('h2');
+    work1b.className = 'article_text';
+    h2.innerHTML = cardData.name;
+    h2.classList.add('product-title');
+    work1b.appendChild(h2);
+    const flex1Ul = document.createElement('ul');
+    flex1Ul.className = 'type_dev';
+    // appends 5 li to Ul
+    work1b.appendChild(loadPosition(cardData.items));
+    const projectDescription = document.createElement('p'); // project description element
+    projectDescription.classList.add('product-details');
+    projectDescription.innerHTML = cardData.description;
+    work1b.appendChild(projectDescription);
+    const technologies = document.createElement('ul'); // tecnologies ul
+    technologies.className = 'product-categories';
+    // build technologies li
+    work1b.appendChild(
+      buildTechnologyBloc(cardData.technologies, technologies),
+    );
+    const nav = document.createElement('nav');
+    const buttonSeeProject = document.createElement('button');
+    buttonSeeProject.classList.add('button-project');
+    buttonSeeProject.type = 'button';
+    buttonSeeProject.innerHTML = 'See project';
+    buttonSeeProject.addEventListener('click', () => {
+      showPopupWindow(cardData);
+    });
+    nav.appendChild(buttonSeeProject);
+    work1b.appendChild(nav);
+    rows.appendChild(work1bImg); // append image side to card
+    rows.appendChild(work1b); // append details side to card
+    workSection.appendChild(rows);
+  });
+}
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, please make sure to follow as closely as possible the FIGMA design:

|Template|
|---|
| ![image](https://github.com/peter-mwovi/portfolio_setup/assets/77069011/23fc285a-dda2-41e3-9119-3636ff064162) | 
|Your Design|
| ![imageedit_2_3813003084](https://github.com/peter-mwovi/portfolio_setup/assets/77069011/d069a341-ef61-48c5-9811-4ddc5500becd) |


- [ ] 🚧 The font size isn't the right one. Please consider increasing it! 🤓 
- [ ] 🚧 The buttons' content should be center aligned, as shown in the design. Please fix it! 🤓 
- [ ] 🚧 According to the design, the background should be blurred when the pop-up is open. Please implement this! 🤓 
- [ ] 🚧 **[OPT]:** Currently, when the pop-up is open, the user can scroll through the page in the background. This isn't great for user experience. It would be nice if you can prevent this behavior from happening when the pop-up is open. 🤓 

For the mobile version:

|Template|Your design|
|---|---|
| ![image](https://github.com/peter-mwovi/portfolio_setup/assets/77069011/a42b1bd9-c467-4343-859a-26cc90d89c3a) | ![image](https://github.com/peter-mwovi/portfolio_setup/assets/77069011/55c0b8fd-c2ef-4fe4-945d-e5250654b215) |

- [ ] 🚧 Please consider using the same close icon as shown in the Figma, you can download the asset. 🤓 
- [ ] 🚧 Also consider fixing the button's alignment. 🤓